composer.json

```diff
@@ -6,8 +6,9 @@
     "readme": "./README.md",
     "license": "Apache-2.0",
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php-64bit": "^7.4 || ^8.0",
```

Comment:
Per the Slack thread it sounds like this should be a runtime check

Reply:
yeah, that makes sense - 32bit is probably more prevalent than I suspected

Reply:
Updated to throw an exception here. I believe this will be unhandled rn, but if it's preferable to just log here after stopping to do things upstream I can add that in too

Reply:
Unhandled is fine. We can later make some of the factories deal with handling that exception.